bindings/python/CHANGELOG.rst

```diff
@@ -1,11 +1,13 @@
 Changelog
 =========
 
-Changes in Version 1.1.0b0
---------------------------
+Changes in Version 1.1.0
+------------------------
 
 - Add support for Azure and GCP KMS providers.
-- Bundle libmongocrypt 1.1.0-beta1 in release wheels.
+- Add support for temporary AWS credentials via the "sessionToken" option.
```

Comment:
I think this entry should reflect the fact that `sessionToken` is only supported when binding libmongocrypt 1.2 or later even though pymongocrypt 1.1 supports libmongocrypt 1.1+:
```
- Add support for temporary AWS credentials via the "sessionToken" option when using libmongocrypt 1.2 or later.
```
OR
```
- Add support for temporary AWS credentials via the "sessionToken" option (requires libmongocrypt 1.2+).
```